app/views/categories/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="d-flex justify-content-between top text-light p-2">
+  <a href="#" class="text-light"><i class="bi bi-list h1"></i></a>
+  <h1>CATEGORIES</h1>
+  <a href="#" class="text-light"><i class="bi bi-search h1"></i></a>
+</div>
+<ul class="top d-flex justify-content-center gap-2 pb-2 list-unstyled">
+  <li><%= link_to_unless_current("Most recent", root_path ) %><li>
+  <li><%= link_to_unless_current("Most ancient",  older_categories_path ) %><li>
+</ul>
+
+<% if !@categories.empty? %>
+  <div id="categories" class="p-2">
+  <% @categories.each do |category| %>
+    <div class="category d-flex border mb-1 p-1">
+      <div class="logo me-2 col-3" style="background-image: url(<%= category.icon %>)"></div>
+      <div class="col-5">
+        <%= link_to category_path(category), class: 'text-decoration-none' do %>
+          <%= category.name %>
+        <% end %>
+        <p><%= category.created_at.strftime("%d %h %Y")  %></p>
+      </div>
+      <p></p>
+      <% @total = 0 %>
+      <% category.deal_categories.each { |e| @total += e.deal.amount }%>
+      <p class="text-end col-3">Total: $<%= @total %></p>
+    </div>
+  <% end %>
```

Comment:
- [ ] Please make sure to adjust the design for the `categories` and `deals` index pages as in the design guidelines. Please see the link in the comment above.
![Screenshot (1)](https://github.com/alicemirigo92/Budget-App/assets/47625407/7f238b0b-cc9d-433b-8e4f-d4c66ef7c038)